src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -16,6 +16,8 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
 {
+    using System;
```

Comment:
Should be outside the namespace.

Reply:
Yes, sadly this is where "Add using" put it :(

Reply:
Filed a bug: https://github.com/dotnet/roslyn/issues/23586

Reply:
Sorted.